### DISCUSSION TITLE
No Install Locations Working

### DISCUSSION THREAD

**Discussion by DARKINT3NS3TY** [flybywiresim/aircraft#3823] [Help]
Hi All,

I need help with installation.
I have tried both the downloader and the manual method, neither seem to work.
I have also tried multiple different locations to install the file to, but none seem to work. 

I have MSFS steam edition, with my steam games installed onto my D: drive and not my C: drive.
I have gone into steam to locate where my MSFS is installed. I then try to install the A32nx into this location but nothing comes up in MSFS.

If i look in the content manager, i can see the A32nx, but can't do anything with it. The A32nx does't come up in my library of aircrafts.

If anyone can help out it would be great! I really want to fly this thing!

MSFS Installation location on my PC:
D:\Programs\Steam\steamapps\common\MicrosoftFlightSimulator

Where the downloader initially tried to install the A32nx:
D:\Microsoft Flight Simulator\Community

Please Help

**Comment by Valastiri** [comment]
Please check what folder MSFS is tracking for community installations. 

Steps:

- Settings -> turn on Dev Tools
- Go to dev tools in the top bar and check for virtual file system. 

That should tell you what community folder is being tracked. (This is regardless of what your installer tries to detect you need to verify it from within MSFS). 

Visit us on discord if you continue to. have issues.
